k8s/helmfile/env/local/minio.values.yaml.gotmpl

```diff
@@ -0,0 +1,19 @@
+existingSecret: minio-credentials
+
+image:
+  tag: RELEASE.2023-07-07T07-13-57Z
+
+mode: standalone
+
+replicas: 1
+
+persistence:
+  size: 4Gi
```

Comment:
This seems very generous, but I also have the newest laptop around here. Should this be lower?

Reply:
I feel like this is fine; not that much space really and even slow laptops may have more ssd